src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  justify-content: center;
+  place-items: center;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
```

Comment:
Confirm that the grid layout for a minimum width of 1024px aligns with the design requirements. Make necessary changes to the number of columns and their sizes if the current settings do not match the design.